otherlibs/stdune/src/path.mli

```diff
@@ -230,6 +236,32 @@ type t = private
 
 include Path_intf.S with type t := t
 
+module Table : sig
```

Comment:
Why couldn't this be `module Table : Hashtbl.S with type key = path`? (Whatever the answer is, I think it's worth documenting it as a comment.)

Reply:
Will add a comment. The reason is that it would require implementing more functions than we actually use. Not a great concern, but it does more code that is effectively dead and might make us rely on more properties of a regular hash tables than might want to.

Reply:
Right, agreed, it seems best not to maintain dead code.